packages/engine-server/src/doctor/service.ts

```diff
@@ -330,6 +333,31 @@ export class DoctorService implements Disposable {
         );
         return { exit: true };
       }
+      case DoctorActionsEnum.FIX_AIRTABLE_METADATA: {
+        // Converts the airtable id in note frontmatter from a single scalar value to a hashmap
+        if (!podId) {
+          assertInvalidState(
+            "Please provide pod Id to perform Doctor operation"
+          );
+        }
+        doctorAction = async (note: NoteProps) => {
+          //get airtable id from note
+          const airtableId = _.get(note.custom, "airtableId") as string;
```

Comment:
I think this should check if the note has an airtable ID already, and skip notes that don't have them. Looks likewe do that filtering in the plugin code, but I think we can do it here instead so the CLI code also benefits from it.

Reply:
updated doctor/service.ts to filter candidates.